cross-modal-search/pods/index-image-kv.yml

```diff
@@ -1,30 +1,31 @@
-!BinaryPbIndexer
-with:
-  index_filename: images.gz
-metas:
-  name: images_kv_idx  # a customized name
-  workspace: './workspace'
-requests:
-  on:
-    IndexRequest:
-      - !FilterQL
-        with:
-          lookups: {'modality': 'image'}
-      - !Buffer2URI
-        with:
-          base64: True
-      - !ExcludeQL
-        with:
-          fields:
-            - chunks
-            - embedding
-            - buffer
-      - !KVIndexDriver
-        with:
-          executor: images_kv_idx
-    SearchRequest:
-      - !KVSearchDriver
-        with:
-          executor: images_kv_idx
-          traversal_paths: ['m']
-          recur_on: matches
+#!BinaryPbIndexer
```

Comment:
We don't use any of this?

Reply:
The logic is adapted for 2.0 in executors.py